spec/numberformat.html

```diff
@@ -457,10 +457,11 @@
       <emu-alg>
         1. Let _nf_ be *this* value.
         1. If Type(_nf_) is not Object, throw a *TypeError* exception.
-        1. If _nf_ does not have a [[boundFormat]] internal slot, throw a *TypeError* exception.
-        1. If _nf_.[[boundFormat]] is *undefined*, then
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
+        1. If _nf_.[[initializedNumberFormat]] is *undefined*, then
```

Comment:
as for #57, nothing will change in the algos, just a non-normative note that will be clarify!